pgsmo/objects/table/templates/+default/get_relations.sql

```diff
@@ -8,7 +8,7 @@
 SELECT c.oid, quote_ident(n.nspname)||'.'||quote_ident(c.relname) AS like_relation
 FROM pg_class c, pg_namespace n
 WHERE c.relnamespace=n.oid
-    AND c.relkind IN ('r', 'v', 'f')
+    AND c.relkind IN ('r', 'v', 'f', 'p')
```

Comment:
I included `'p'` in [relkind](https://www.postgresql.org/docs/current/catalog-pg-class.html) in a few queries that weren't strictly required for the functionality in this PR.

Perhaps I should revert that. However, I'm assuming those queries should gather information about partitioned tables as well in case it it's useful.